.github/workflows/publish-cli.yml

```diff
@@ -12,11 +12,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
-      - name: Install JDK
+      - name: Install JDK 11
```

Comment:
We would need to stick with Java 8 due to requirements in our Android modules

Reply:
Ok I will revert these changes and will do a follow up PR for the java 11 update. It also is required when Android Gradle Plugin is updated to 7.4.0 or higher.
[Android Gradle Plugin](https://developer.android.com/studio/releases/gradle-plugin#compatibility-7-4-0)